### DISCUSSION TITLE
file size unit

### DISCUSSION THREAD

**Discussion by r3dge** [lxqt/lxqt#2563] [Q&A]
Hi there,

Currently pcmanfm-qt is displaying (in list mode) the file size using this unit : KiB,MiB,GiB,TiB
Is there a way to display the size in a more "user friendly" size : KB, MB,GB,TB ?

Thanks for your support !

Regards,

**Comment by tsujan** [comment]
I don't know how `KB` could be considered "more user friendly", because it isn't a measure of weight or length ;) But the answer to your question is "no".

**Comment by r3dge** [comment]
What I meant is that very few users know this unit of measurement which is more used by experts than basic users.
Thanks for your answer.
